CONTRIBUTING.md

```diff
@@ -33,37 +33,54 @@ as well as other packages in the monorepo.
 ### Create a new lerna package
 
 ```shell
-$ lerna create truffle-mycmd
+$ lerna create mycmd
 ```
 
 ### Add the package to `@truffle/core`
 
 ```shell
-$ lerna add truffle-mycmd --scope=@truffle/core
+$ lerna add mycmd --scope=@truffle/core
 ```
 
 ### Create a new command in `@truffle/core`
 
-Create a new file in `packages/core/lib/commands/`, let's call it `mycmd.js`.
+1. Create a new directory in `packages/core/lib/commands/`, let's call it `mycmd`.
+2. Create 3 javascript files inside the **mycmd** directory with the filenames **run.js**, **meta.js** and **index.js**:  
+    * **run.js** contains the entry function after a user calls your command. The entry function has to be anonymous and of type async.
```

Comment:
Strictly speaking the main function for a command doesn't _need_ to be `async` but may be if it needs to be!

Reply:
Will change that as soon as Im home

Reply:
Resolved with commit 7f600e216ec2066414780582ff06e058a91730e5